tests/sources/test_jira.py

```diff
@@ -453,8 +586,8 @@ async def test_ping_for_failed_connection_exception(client_session_get):
 @pytest.mark.asyncio
 async def test_validate_config_for_ssl_enabled_when_ssl_ca_empty_raises_error():
     """This function test _validate_configuration when certification is empty when ssl is enabled"""
-    async with create_jira_source() as source:
-        source.configuration.get_field("ssl_enabled").value = True
+
+    async with create_source(JiraDataSource, ssl_enabled=True) as source:
```

Comment:
Instead of calling `create_source` here, you could create an arg in `create_jira_source` for `ssl_enabled` so that you can still initiate all of the missing configuration values. Otherwise your configuration is initialised here empty except for `ssl_enabled`.

e.g.

```
async def create_jira_source(ssl_enabled=False):
```

and then call it like

```
 create_jira_source(ssl_enabled=True)
```